acoular/spectra.py

```diff
@@ -294,8 +300,17 @@ class PowerSpectra(BaseSpectra):
 
     @property_depends_on('_source.numsamples, block_size, overlap')
     def _get_num_blocks(self):
+        if self._num_blocks != 0: # The source is a real time stream
```

Comment:
This could also be used for non real time data.

Reply:
Resolved in https://github.com/acoular/acoular/pull/285/commits/e156e01325b09ece2aa8ac2ba9572accd96e24af as proposed